src/server/multi_test.cc

```diff
@@ -396,13 +396,13 @@ TEST_F(MultiTest, Eval) {
   resp = Run({"eval", "return redis.call('get', 'foo')", "0"});
   EXPECT_THAT(resp, "42");
 
-  Run({"script", "flush"});
+  Run({"script", "flush"});  // Reset global flag from autocorrect
```

Comment:
what do you mean by this?

Reply:
this is a very fragile test setup, because when we call 

>  resp = Run({"eval", "return redis.call('get', 'foo')", "0"});
  EXPECT_THAT(resp, ErrArg("undeclared"));

with out autocorrect flag, the whole script is automatically transferred to global mode, so all following tests would run the same sha in global mode  (we reuse redis.call(get, foo)) 

So it's a comment why we run script flush